packages/iov-core/src/signingservercore.ts

```diff
@@ -9,29 +9,57 @@ import { UserProfile, WalletId } from "@iov/keycontrol";
 
 import { MultiChainSigner } from "./multichainsigner";
 
+export interface GetIdentitiesAuthorization {
```

Comment:
I like this design, called after checking request against what we will show.

I would like to add a real identifier, like "session", so we can keep track of who is asking for what. But that is another issue.